sage-ios/SAGE/SAGE/AdminOperations.swift

```diff
@@ -21,6 +21,9 @@ class AdminOperations {
                 let user = User(propertyDictionary: userDict as! [String: AnyObject])
                 userArray.append(user)
             }
+            userArray.sortInPlace({ (user1, user2) -> Bool in
```

Comment:
make the { (user1, user2) -> Bool in user1.firstName < user2.firstName } part into a function, not the whole sortInPlace part